tests/test_l2_basic.py

```diff
@@ -711,3 +714,151 @@ def test_l2_mac_move_1(npu, dataplane):
             npu.set(npu.port_oids[idx], ["SAI_PORT_ATTR_PORT_VLAN_ID", npu.default_vlan_id])
         
         npu.remove(vlan_oid)
+
+
+def test_l2_db_aging_test(npu, dataplane):
+    """
+    Description:
+    Check access to access VLAN members forwarding
+    Test scenario:
+    1. Create a VLAN 10
+    2. Add three ports as untagged members to the VLAN
+    3. Send packet from port1 to port2 and verify on each of ports
+    4. Send packet from port2 to port1 and verify only on port1
+    5. Send packet from port2 to port1 and verify on each of ports
+    6. Clean up configuration
+    """
+    vlan_id = "10"
+    macs = ['00:11:11:11:11:11', '00:22:22:22:22:22']
+    max_port = 3
+    fdb_aging_time = 10
+    vlan_mbr_oids = []
+
+    vlan_oid = npu.create(SaiObjType.VLAN, ["SAI_VLAN_ATTR_VLAN_ID", vlan_id])
+
+    for idx in range(max_port):
+        npu.remove_vlan_member(npu.default_vlan_oid, npu.dot1q_bp_oids[idx])
+        vlan_mbr = npu.create_vlan_member(vlan_oid, npu.dot1q_bp_oids[idx], "SAI_VLAN_TAGGING_MODE_UNTAGGED")
+        vlan_mbr_oids.append(vlan_mbr)
+        npu.set(npu.port_oids[idx], ["SAI_PORT_ATTR_PORT_VLAN_ID", vlan_id])
+
+    try:
+        if npu.run_traffic:
+            pkt = simple_tcp_packet(eth_dst=macs[1],
+                                    eth_src=macs[0],
+                                    ip_dst='10.0.0.1',
+                                    ip_id=101,
+                                    ip_ttl=64)
+
+            pkt1 = simple_tcp_packet(eth_dst=macs[0],
+                                     eth_src=macs[1],
+                                     ip_dst='10.0.0.1',
+                                     ip_id=101,
+                                     ip_ttl=64)
+
+            send_packet(dataplane, 0, pkt)
+            verify_each_packet_on_each_port(dataplane, [pkt, pkt], [1, 2])
+
+            send_packet(dataplane, 1, pkt)
+            verify_packets(dataplane, pkt, [0])
+            time.sleep(fdb_aging_time + 2)
+
+            send_packet(dataplane, 1, pkt)
+            verify_each_packet_on_each_port(dataplane, [pkt1, pkt1], [0, 2])
+
+    finally:
+        for idx in range(max_port):
```

Comment:
You should flush FDB entries before vlan members removal otherwise it will fail.